vm-virtio/src/mem.rs

```diff
@@ -686,29 +688,41 @@ impl MemEpollHandler {
                                 config.usable_region_size =
                                     cmp::max(config.usable_region_size, tmp_size);
                                 if let Err(e) = self.signal(&VirtioInterruptType::Config) {
-                                    need_break = true;
+                                    signal_error = true;
                                     Err(Error::ResizeTriggerFail(e))
                                 } else {
                                     Ok(())
                                 }
                             };
                             if let Err(e) = &r {
-                                error!("{:?}", e);
+                                // This error will send back to resize caller.
+                                error!("Handle resize event get error: {:?}", e);
```

Comment:
Error doesn't get returned in this case.

Reply:
This error will be send back to resize caller.  And be handled in the caller side.

Reply:
Ah, very unusual got it.